src/app/app.module.js

```diff
@@ -7,14 +7,16 @@ angular.module('esn.inbox-calendar', [
   'esn.calendar.libs',
   'esn.resource.libs',
   'linagora.esn.videoconference',
-  'linagora.esn.videoconference.calendar'
+  'linagora.esn.videoconference.calendar',
+  'esn.lodash-wrapper'
```

Comment:
We don't use use this anymore. If you see lodash injected as an AngularJS dependency, remove it.

Reply:
Fixed